store/etcd/rulesets_test.go

```diff
@@ -153,6 +153,114 @@ func TestList(t *testing.T) {
 	})
 }
 
+func TestListPaths(t *testing.T) {
```

Comment:
I think that this is implicit? Not sure to understand clearly what do you mean?

Reply:
Because test names have to, to an extent, be terse, it's useful to describe the desired behaviour we're testing for.  One of the roles of tests is to act as a kind of documentation for what the code *should* do.  When we have a a big test case like this, with multiple sub-tests it's probably worth spending a little extra time to summarise what it is we expect.

Reply:
@tealeg agreed! Thank you for the deep explanation. 🙏

Reply:
OK, this is moving very much in the right direction.  One small tweak: it's not normally necessary to say in the text that "this tests...", because it is after all a test.  It's nicer to write a positive statement.  A more concrete example, say we're testing the `+` function.  We could write:

``
// TestPlus tests that plus returns the sum of two numbers.
func TestPlus(t *testing.T) {
```

.. but its better to write: 
```
// + returns the sum of two numbers
func TestPlus(t *testing.T) {
```

Reply:
Oh okay, got it. I will try again.

Reply:
:+1:

Reply:
:+1: